common/util/util.go

```diff
@@ -79,6 +82,14 @@ func FetchPublicIP() string {
 	return string(body)
 }
 
+func FetchNodeName() (string, error) {
```

Comment:
Avoid calling LookupEnv multiple times, if MY_NODE_NAME is not available, exit the process directly.

```suggestion
var nodeName string

func FetchNodeName() string {
	if nodeName != "" {
		return nodeName
	}
	nodeName, isExist := os.LookupEnv(meshConstants.MY_NODE_NAME)
	if !isExist {
		klog.Exitf("env %s not exist", meshConstants.MY_NODE_NAME)
	}
	return ""
}
```

Reply:
Thanks for the suggestions. Done.
in addition,  I also changed edgemesh-server to use common/util to get nodeName.